### PR TITLE
Fix #3188 and some profile display improvments

### LINKED ISSUE #3188: profile is not displayed when drawing new line feature
Situation:
* open map.geo.admin.ch -> draw & measure
* draw a new line
* click on tab profile

the profile tab is empty.
image

 If you add an additional vertex it will be drawn.
image

issue has been reported by customer via webgis mailbox.

### PR DESCRIPTION
**Pull request by oterral**
Fixes #3188  and the same bug for measure directive.

Other things:

  -  introduces promises in ProfilService
  -  bigger profile for measure features
  -  no more text truncated (name on top right and Y value on top left)
  -  fixes reducing of size on each appearance of profile.

[Test](https://mf-geoadmin3.dev.bgdi.ch/teo_prof)

### CODE REVIEW

File: src/components/measure/MeasureDirective.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ goog.require('ga_measure_service');
                 update(evt.target);
               });
             });
-            newFeature.changed();
+            update(newFeature);
```
Comment (gjn) [inline src/components/measure/MeasureDirective.js]: strange that this stopped working with an ol3 update. @oterral @fredj Do you know the reason?

Comment (gjn) [inline src/components/measure/MeasureDirective.js]: This alone would have fixed the bug.

**Comment by gjn** [comment]
Thanks alot. Nice improvement.

I'll cherry-pick this in the release branch and deploy it (afternoon).

File: src/components/measure/MeasureDirective.js (hunk 1 above)
Comment (oterral) [inline src/components/measure/MeasureDirective.js]: don't know  why the first call to `changed()` is ignored, the 2nd trigegrs correctly the event.

Comment (gjn) [inline src/components/measure/MeasureDirective.js]: probably timing issue. We bind to `change` event just before, then try to call it.

Comment (fredj) [inline src/components/measure/MeasureDirective.js]: @gjn no clue ... do you use the latest ol3 version? https://github.com/openlayers/ol3/releases/tag/v3.14.2

**Pull request merged**